orc8r/gateway/python/magma/magmad/config_manager.py

```diff
@@ -145,4 +146,29 @@ def did_mconfig_change(serv_name):
             if srv in mconfig.configs_by_key:
                 configs_by_key[srv] = mconfig.configs_by_key.get(srv)
 
+        # version should be in 1.1.1 format with only positive numbers allowed
```

Comment:
I would pull all the versioning stuff into a separate function with some descriptive name (`_warn_if_versions_are_incompatible`? Or something nicer :) I've never been good with naming)